core/src/com/po/fuck/weapons/Bullet.java

```diff
@@ -29,6 +29,9 @@ public void draw(CenterDrawer drawer) {
         sprite.setRotation(-velocity.angleDeg());
         drawer.draw(sprite, position);
     }
+    protected boolean canDamage(Entity entity) {
```

Comment:
I think it is better to use `tryDamage`, because for now in every place when we are checking if we can damage, we are doing damage afterwards

Reply:
@MAKMED1337 Is that nice idea to give all bullets ```initialDamage``` and ```periodicalDamage``` so damage will look like ```initialDamage + periodicalDamage * delta```?

Reply:
For now we do not have a combined type of damage, so I think only storing damage as "DPS" is ok, we can pass `float damage` as a parameter to the `tryDamage`.